llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -473,14 +475,11 @@ AArch64TargetLowering::AArch64TargetLowering(const TargetMachine &TM,
     }
   }
 
-  if (Subtarget->hasSVE2p1() || Subtarget->hasSME2()) {
-    addRegisterClass(MVT::aarch64svcount, &AArch64::PPRRegClass);
-    setOperationPromotedToType(ISD::LOAD, MVT::aarch64svcount, MVT::nxv16i1);
-    setOperationPromotedToType(ISD::STORE, MVT::aarch64svcount, MVT::nxv16i1);
+  setOperationPromotedToType(ISD::LOAD, MVT::aarch64svcount, MVT::nxv16i1);
+  setOperationPromotedToType(ISD::STORE, MVT::aarch64svcount, MVT::nxv16i1);
 
-    setOperationAction(ISD::SELECT, MVT::aarch64svcount, Custom);
-    setOperationAction(ISD::SELECT_CC, MVT::aarch64svcount, Expand);
-  }
+  setOperationAction(ISD::SELECT, MVT::aarch64svcount, Custom);
+  setOperationAction(ISD::SELECT_CC, MVT::aarch64svcount, Expand);
```

Comment:
Can you move these into an existing `isSVEorStreamingSVEAvailable` setOperationAction block?

Reply:
I've moved these close to where actions for other predicate types are set, if that's what you meant.